applications/crossbar/doc/tasks.md

```diff
@@ -110,11 +119,15 @@ curl -v -X GET \
 }
 ```
 
+#### Add a new system task
+
 
 #### Add a new task
 
 > PUT /v2/accounts/{ACCOUNT_ID}/tasks
 
+Note: There are tasks that run against system resources, only for use by the superduper admin (like rate uploading), which can omit `/accounts/{ACCOUNT_ID}` from the URI. Leaving the account in the URI should have no impact.
+
```

Comment:
Other tasks will profit from this:

1. Move the "is super admin" check logic into `kz_tasks:new`
1. Really document this `/v2/tasks` endpoint (curl PUT with success & error cases)
1. Add a boolean `<<"needs_admin">>` to tasks requiring superadmin rights to be run, where `true` means returning `{error, unauthorized}` when not a superadmin in `kz_tasks:new`.
1. Update tasks with `is_allowed/1` code to only rely on that boolean instead

Because as it is this /v2/tasks API makes no sense. Plus, it violates the `extra_args()` type.